falcon/routing/converters.py

```diff
@@ -83,15 +86,41 @@ def convert(self, value):
         except ValueError:
             return None
 
+        return self.validate_min_max_value(value)
+
+    def validate_min_max_value(self, value):
```

Comment:
Maybe this method could be made private? Like `_validate_min_mix(value)`? Unless we want to document it and make it official (I don't think so).